docs/book/src/topics/failure-domains.md

```diff
@@ -128,6 +128,21 @@ spec:
 
 ```
 
+If you can't use `Machine` (or `MachineDeployment`) to explicitly place your VMs (for example, there is a `KubeAdmControlPlane` does not accept those as an object reference but rather uses `AzureMachineTemplate` directly), then you can opt to restrict the announcement of discovered failure domains from the cluster's status itself.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureCluster
+metadata:
+  name: my-cluster
+  namespace: default
+spec:
+  location: eastus
+  failureDomains:
+    fd1:
```

Comment:
is fd1 actually what an Azure fd id would look like?

Reply:
when we were testing it on dev cluster, the values reported in cluster status were `1`, `2` etc. but i wasn't sure if that's the standard or not and for some reason it felt more explicit adding the `fd`

changed to `1:`